### PR TITLE
Integrate Alexa Skill and List Events

### PR DESCRIPTION
**Pull request by wuelcas**
Integrate request handlers for [Alexa Skill Events](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/skill-events-in-alexa-skills#skill-events-in-json-format) and [Alexa List Events](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/list-events-in-alexa-skills). Update documentation and account linking example

### CODE REVIEW

**Comment by armonge** [comment]
@wuelcas seems like one of the sample projects is failling, can you please look at that and fix the issue? Perhaps some dependency was updated in npm and is no longer compatible with some node version or similar

**Comment by armonge** [comment]
voxa and it's sample projects should support the node versions available in aws lambda

**Comment by wuelcas** [comment]
I was thinking about that, I'll check that out

File: docs/statemachine-skill.rst [hunk 1]
```diff
@@ -243,3 +243,67 @@ Handle events from the `AudioPlayer interface <https://developer.amazon.com/publ
...
+
+Handle request for the `Alexa Skill Events <https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/skill-events-in-alexa-skills#skill-events-in-json-format>`_
+
+.. js:function:: alexaSkillEventCallback(alexaEvent)
+
+  All the alexa skill event callbacks get a :ref:`alexa event <alexa-event>` and a :ref:`reply <reply>` object
+
+  :param AlexaEvent alexaEvent: The :ref:`alexa event <alexa-event>` sent by Alexa
+  :param object reply: A reply to be sent as the response
+  :returns object reply: Alexa only needs an acknowledgement to you received and processed the event so it doesn't need to resend the event. Just returning the :ref:`reply <reply>` object is enough
```
Comment (omenocal) [inline docs/statemachine-skill.rst]: `acknowledgement that you`

File: docs/statemachine-skill.rst [hunk 2]
```diff
@@ -243,3 +243,67 @@ Handle events from the `AudioPlayer interface <https://developer.amazon.com/publ
...
+
+Handle request for the `Alexa List Events <https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/list-events-in-alexa-skills#list-events-json>`_
+
+.. js:function:: alexaListEventCallback(alexaEvent)
+
+  All the alexa list event callbacks get a :ref:`alexa event <alexa-event>` and a :ref:`reply <reply>` object
+
+  :param AlexaEvent alexaEvent: The :ref:`alexa event <alexa-event>` sent by Alexa
+  :param object reply: A reply to be sent as the response
+  :returns object reply: Alexa only needs an acknowledgement to you received and processed the event so it doesn't need to resend the event. Just returning the :ref:`reply <reply>` object is enough
```
Comment (omenocal) [inline docs/statemachine-skill.rst]: `acknowledgement that you`

File: samples/accountLinking/skill/states.js [hunk 3]
```diff
@@ -12,4 +12,18 @@ exports.register = function register(skill) {
...
+    const userId = alexaEvent.user.userId;
+    const timestamp = alexaEvent.request.timestamp;
+    console.log(`skill was enabled for user: ${userId} at ${timestamp}`);
+    return reply;
+  });
+
+  skill['onAlexaSkillEvent.SkillAccountLinked']((alexaEvent, reply) => {
+    const userId = alexaEvent.user.userId;
+    const accessToken = alexaEvent.request.body.accessToken;
+    console.log(`User: ${userId} link his account. Access Token is ${accessToken}`);
```
Comment (omenocal) [inline samples/accountLinking/skill/states.js]: `linked`

**Pull request merged**